universal-application-tool-0.0.1/app/views/style/AdminStyles.java

```diff
@@ -51,4 +51,28 @@
           Styles.OPACITY_0,
           Styles.TOP_0,
           Styles.W_FULL);
+
+  public static final String BODY =
+      StyleUtils.joinStyles(
+          BODY_GRADIENT_STYLE,
+          Styles.BOX_BORDER,
+          Styles.H_SCREEN,
+          Styles.W_SCREEN,
+          Styles.OVERFLOW_HIDDEN,
+          Styles.FLEX);
+
```

Comment:
suggestion (non-blocking): Can you add a brief comment on CENTERED and FULL for what kind of elements they should be used on and roughly what they do visually? Maybe rename them `MAIN_CENTERED` and `MAIN_FULL` if they should only be used on a `main` tag.